libnmstate/state.py

```diff
@@ -679,6 +717,39 @@ def merge_route_rules(self, other_state):
             }
 
 
+def _get_bond_slaves(ifstate):
+    lag = ifstate.get(Bond.CONFIG_SUBTREE, {})
+    return lag.get(Bond.SLAVES, [])
+
+
+def _get_linux_bridge_slaves(ifstate):
```

Comment:
The `appliers/linux_bridge.py` already has `get_slaves_from_state()`.

I don't like the idea of placing everything related to state to `state.py`. If possible, please
create `iface_state/bridge.py`, `iface_state/ovs.py` and `iface_state/bond.py` files
to hold these interface type specific state related functions.
Not a must, if you don't want it, just remove the duplciation of `_get_linux_bridge_slaves()`
is enough for this PR.

Reply:
And in `metadata.py` we have:

```
59:        get_slaves_func=_get_bond_slaves_from_state,
60:        set_metadata_func=_set_common_slaves_metadata,
66:        get_slaves_func=_get_ovs_slaves_from_state,
73:        get_slaves_func=linux_bridge.get_slaves_from_state,
```

Reply:
I posted https://github.com/nmstate/nmstate/pull/797 to allow re-using the same getters.

Reply:
Done